scaladoc/src/dotty/tools/scaladoc/tasty/comments/markdown/DocFlexmarkExtension.scala

```diff
@@ -47,33 +45,17 @@ object DocFlexmarkParser {
     }
 }
 
-case class DocFlexmarkRenderer(renderLink: (DocLink, String) => String, snippetCheckingFunc: SnippetChecker.SnippetCheckingFunc)
+case class DocFlexmarkRenderer(renderLink: (DocLink, String) => String)
   extends HtmlRenderer.HtmlRendererExtension:
-
     def rendererOptions(opt: MutableDataHolder): Unit = () // noop
 
-    object FencedCodeBlockHandler extends CustomNodeRenderer[ast.FencedCodeBlock]:
```

Comment:
Doesn't moving snippet compilation to the parser mean that I can't get the compilation messages now?

Reply:
There is couple of ways to get compilation messages e.g.:
- we can store them in member
- we can create custom node and keep messages there